pages/ksphere/kommander/latest/operations/cloud-providers/index.md

```diff
@@ -156,6 +156,69 @@ Figure 7 - Cloud Provider Form with values
 
 Once created, a Cloud Provider’s display name or credentials can be updated.
 
+### Configuring an Azure cloud provider
+
+When creating an Azure cloud provider, you need an [ansible inventory file](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html) and ssh keys.
+
+#### Create a new set of credentils via CLI commands
+
+```
+az login
+```
+
+```
+az role assignment cerate --assignee YOUR_USER_LOGIN --role "User Access Administrator"
+```
+
+```
+az account set --subscription=SUBSCRIPTION_ID"
+```
+
+```
+az ad sp create_for_rbac --role="Contributor" --scopes="/subscriptions/SUBSCRIPTION_ID"
+```
+
+The output should look similar to this:
+
+```
+{
+    "appId": "<APP_ID>",
+    "displayName": "azure-cli-...",
+    "name": "http://azure-cli-...",
+    "password": "<PASSWORD>",
+    "tenant": "<TENANT>"
+}
+```
+
+#### Fill out the rest of the form
+
+- Fill out a display name for your cloud provider that you can reference later.
```

Comment:
The usage of periods at the end of bullet points is not regular. To `.` or not to `.`, that is the question 🤔 .

Reply:
I'll agree with that. A rule of thumb is if you have a group of bullets, they should be consistent. Either have periods or not. Usually a complete sentence warrants a bullet.

Reply:
I added bullets.